index.html

```diff
@@ -0,0 +1,265 @@
+          </li>
+          <li class="single-cont-author">
+            <a href="#" class="link-nav">Logout</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="cont-navbar">
+        <div class="navbar-wrapper">
+          <a href="./index.html" class="link-page">
+            <figure class="logo-img-cont">
+              <img
+                src="./images/logo.png"
+                alt="logo"
+                class="logo-footer logo-navbar"
+              />
+              <div class="logo-text">
+                <h1 class="website-name logo-name">Earth</h1>
+                <h2 class="website-name logo-name">Planet</h2>
+              </div>
+            </figure>
+          </a>
+          <nav class="link-nav-cont">
+            <ul class="cont-list">
+              <li class="single-cont-list">
+                <a href="./about.html" class="link-nav-page">About</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#main" class="link-nav-page">Program</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">Join</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">Sponsor</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page">News</a>
+              </li>
+              <li class="single-cont-list">
+                <a href="#" class="link-nav-page action-camp">CC Campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <section class="header-cont">
+      <h1 class="heading">&nbsp;</h1>
+      <div class="intro">
+        <h4 class="intro-salutation">"Hello! Environmental protector"</h4>
+        <h1 class="intro-title">World Environment Summit 2021</h1>
+        <p class="intro-desc">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum,
+          incidunt delectus optio illo ducimus vero recusandae minus doloremque
+          ex repellat deserunt ipsa cupiditate quos, dolore facere voluptas
+          facilis non reiciendis?
+        </p>
+        <h2 class="intro-date">2021.10.11(THU) ~ 16(FRI)</h2>
+        <h5 class="intro-location">
+          @ Lorem, ipsum dolor sit amet consectetur adipisicing.
+        </h5>
+      </div>
+    </section>
+
+    <section class="main" id="main">
+      <h1 class="heading">&nbsp;</h1>
+      <div class="main-wrapper">
+        <h1 class="main-title">Main Program</h1>
+        <ul class="activity-cont">
+          <li class="single-activity-wrapper">
+            <ul class="activity-prog-cont">
+              <li class="activity-prog">
+                <i class="fas fa-sun fa-3x"></i>
+              </li>
+              <li class="activity-prog">
+                <h2 class="activity-prog-title">Global Warming</h2>
```

Comment:
- [ ] Based on design guidelines this header should have red color

Yours:


![Screenshot from 2021-10-15 19-24-40](https://user-images.githubusercontent.com/62705947/137528432-8df37043-1dda-462a-815c-d5ec027b6631.png)
Original:
![Screenshot from 2021-10-15 19-24-34](https://user-images.githubusercontent.com/62705947/137528458-035ab062-4c57-4b1c-bd1e-44887bb06efb.png)

Reply:
- [ ] In the mobile version the picture and text alignment should stretch the same across each of them

Yours:
![Screenshot from 2021-10-15 19-41-43](https://user-images.githubusercontent.com/62705947/137530382-84c36f79-78df-44e0-8d73-76b904366886.png)
Original:
![Screenshot from 2021-10-15 19-39-48](https://user-images.githubusercontent.com/62705947/137530411-eaa27631-5138-47d4-bc0b-320844526e16.png)